### ISSUE TITLE
Hyperparameter of the training process didn't match the setting of other papers.

### ISSUE THREAD

**Issue by semi-supervised-paper** [benathi/fastswa-semi-sup#1]
Hi,
I notice that the pytorch version of MeanTeacher has different setting of parameters compared to the tensorflow version. 

For example, the learning rate is 0.1 other than 0.003, which is used in other methods, like temporal ensembling. The rampup epoch and rampdown epoch are different from the tensorflow version too. But I notice that learning rate = 0.003, rampup epoch = 80, rampdown epoch = 50 is used in many other semi-supervised methods. 

Does it mean that if we use the setting of parameters from tensorflow version, we can't achieve a good accuracy? 

Thanks.
